src/tracing/builder/geth.rs

```diff
@@ -195,12 +195,7 @@ impl GethTraceBuilder {
                     AccountState::from_account_info(db_acc.nonce, db_acc.balance, code);
 
                 // insert the original value of all modified storage slots
-                for (key, slot) in changed_acc.storage.iter().filter(|(_, slot)| slot.is_changed())
-                {
-                    // empty slots are excluded
-                    if slot.previous_or_original_value.is_zero() {
-                        continue;
-                    }
+                for (key, slot) in changed_acc.storage.iter() {
```

Comment:
does this include reads? geth includes reads if not in diff mode

(also i'm assuming `in_default_mode` means not in diff mode)

Reply:
it does, I checked against all transactions provided by @sslivkoff